### ISSUE TITLE
First Character Of Filename Truncated

### ISSUE THREAD

**Issue by ghost** [gpbl/webpack-cleanup-plugin#27]
I recently incorporated `webpack-cleanup-plugin` into a Webpack project I've recently started.

The important part of my Webpack configuration, that which is related to the issue I am having, is shown below:

```javascript
{
  output: {
    // Root directory for all build artifacts.
    path: path.join(process.cwd(), `dist/`),
  }
}
```

The value for `path` contains a trailing slash.

The resulting value of `fromPath` in `getFiles` would be `C:\Users\hbetts\Workspace\application\dist\`, with `C:\Users\hbetts\Workspace\application\` being the directory of my Webpack-managed project.

The function that maps files located at that path, correctly returns files like - ` C:\Users\hbetts\Workspace\application\dist\application.js`

I would expect `webpack-cleanup-plugin` to delete `application.js`.

What happens, though, is on the `p.substr(npath.length + 1)` line the length of my normalized `fromPath`, which is 43 characters, including the trailing slash, is incremented by 1, and then a substring of the full file path is taken, effectively removing the first 44 characters, which happens to be the last slash, plus that one extra character, the letter `a`. 

Therefore I'm left with a substring of `pplication.js`.

Later, when the normalized `fromPath` is joined with the file name (the value of the substring operation), I get `C:\Users\hbetts\Workspace\application\dist\pplication.js`

An alternative approach would be to replace:

```javascript
p.substr(npath.length + 1)
```

With:

```javascript
require('path').basename(p)
```

`basename` returns the expected file name `application.js`.

cc @foxbunny 

Also #23 may actually resolve this issue. I just haven't tried it yet.

**Comment by x-yuri** [comment]
> Also #23 may actually resolve this issue. I just haven't tried it yet.

It totally does. But there's [no easy way][1] to use the fixed version.

[1]: http://stackoverflow.com/questions/42384182/how-do-i-install-fork-of-an-npm-package

**Comment by ghost** [comment]
@x-yuri thankfully I was able to work around the problem by removing the trailing slash from `dist/`. 

Going forward, though, it would be less disruptive to consumers when `webpack-cleanup-plugin` supports the use of trailing slashes after accepting #23.

**Comment by gpbl** [comment]
This should be fixed in v0.5.1 – please comment back if you still have this issue as I can't really test on Windows...
